docs/CODE_REVIEW.md

```diff
@@ -87,17 +87,16 @@ We wouldn’t want anyone being mean to us because of an oversight, mistake, or
 
 We use this list when performing a code review to ensure that all tasks have been completed.
 
-- [ ] begin by deleting any existing copy of the Git repository, running `docker system prune` to remove any cached artifacts from any prior code review
-- [ ] clone a fresh copy of the Git repository (`git clone https://github.com/ustaxcourt/ef-cms.git`), fetch the pull request and name it for the sprint (e.g., `git fetch origin pull/204/head:sprint-43`), and then switch to that branch (e.g. `git checkout sprint-43`)
+- [ ] Fetch the pull request for the sprint (e.g., `git fetch origin pull/204/head:sprint-43`), and then switch to that branch (e.g. `git checkout sprint-43`)
```

Comment:
While we're here, should we change this explicit pull # references with something like `pull/<PR #>/head:<sprint-<sprint #>` ?

Reply:
Sure thing! Updated in a2ed1d4.